starters/next-drupal-starter/pages/examples/pagination/[[...page]].jsx

```diff
@@ -6,7 +6,7 @@ import {
 } from '../../../lib/stores';
 import { BUILD_MODE } from '../../../lib/constants';
 
-import { Paginator } from '@pantheon-systems/nextjs-kit';
+import Paginator from '../../../components/paginator';
```

Comment:
Didn't we just git rid of this?

Reply:
looks like it might have snuck back in a recent PR, you can delete this component and use the `nextjs-kit` one.